apps/client/src/shared/component/Header/Notification/NotificationItem.tsx

```diff
@@ -20,81 +24,96 @@ import {
 } from "./NotificationItem.css";
 
 type NotificationListProps = {
-  profileImg: string;
-  name: string;
+  id: number;
+  problemId: number | null;
+  solutionId: number | null;
+  groupId: number | null;
+  groupImage: string;
+  groupName: string;
   message: string;
-  date: string;
+  createdAt: string;
   isRead: boolean;
-  onClick: () => void;
-  onDelete: () => void;
+  notificationType: NotificationType;
 };
 
 const NotificationListItem = ({
-  onClick,
-  profileImg,
-  name,
+  id,
+  problemId,
+  solutionId,
+  groupId,
+  groupImage,
+  groupName,
   message,
-  date,
+  createdAt,
   isRead,
-  onDelete,
+  notificationType,
 }: NotificationListProps) => {
   const { isActive, ...handlers } = useA11yHoverHandler();
 
+  const router = useRouter();
+
+  const queryClient = useQueryClient();
+  const { mutate: readNotiMutate } = useReadNotiItemMutation(notificationType);
+  const { mutate: deleteMutate } = useDeleteNotiMutation();
+
+  const handleItemClick = () => {
+    if (!isRead) readNotiMutate(id);
+
+    router.push(
+      `/group/${groupId}${problemId ? `/problem-list/${problemId}` : ""}${
+        solutionId ? `/solved-detail/${solutionId}` : ""
+      }`,
+    );
+  };
+
+  const handleItemDelete = () => {
+    deleteMutate(id, {
+      onSuccess: () => {
+        queryClient.invalidateQueries({
+          queryKey: notificationQueryKey.lists(notificationType),
+        });
+      },
+    });
+  };
+
   return (
-    <li
+    <div
       className={containerStyle}
-      aria-label={`${name}님의 알림: ${message}, ${date}`}
+      aria-label={`${groupName}님의 알림: ${message}, ${createdAt}`}
       {...handlers}
     >
-      <div
-        role="button"
+      <button
         className={notificationContentStyle}
-        onClick={(e) => {
-          e.stopPropagation();
-          onClick();
-        }}
-        onKeyDown={(e) => {
-          e.stopPropagation();
-          handleA11yClick(onClick);
-        }}
-        tabIndex={0}
+        onClick={handleItemClick}
+        aria-label={`${groupName}의 알림으로 이동`}
       >
         <div className={profileStyle}>
           <Image
-            src={profileImg || icnNew}
+            src={groupImage || icnNew}
             width={25}
             height={25}
             className={profileImageStyle}
-            alt={`${name}님의 프로필 이미지`}
+            alt={`${groupName}님의 프로필 이미지`}
           />
           <div>
-            <strong className={nameStyle}>{name} </strong>
-            <span className={messageStyle}>{message}</span>
+            <strong className={nameStyle}>{groupName} </strong>
+            <span className={messageStyle({ isRead })}>{message}</span>
           </div>
         </div>
         <div className={dateContainerStyle}>
-          <div className={dotStyle({ isRead })} />
-          <time className={dateStyle} aria-label={date}>
-            {formatDistanceDate(date)}
+          <time className={dateStyle} aria-label={createdAt}>
```

Comment:
```suggestion
          <time className={dateStyle} datetime={createdAt}>
```
`<time>`태그는 `aria-label`이 아니라 `datetime`이라는 속성을 사용해야 해요.

`aria-label`을 사용하면 **길고 긴 전체 시간**을 읽어주게 되고, 
`<time datetime="xyz">`을 쓰면 텍스트인 `n일 전`을 읽어주면서 보조 도구에게 정확한 시간도 알려줄 수 있습니다.

Reply:
time 태그는 알았어도 datetime으로 컴퓨터가 이해할 수 있는 명확한 시간을 제공해줘야 한다는건 처음 알았네요.
좋은 지식 잘 배워갑니다!